build.gradle

```diff
@@ -51,9 +50,9 @@ dependencies {
 		exclude module: 'launchwrapper'
 		exclude module: 'guava'
 	}
-	implementation 'net.fabricmc:tiny-mappings-parser:0.2.2.14'
-	implementation 'net.fabricmc:tiny-remapper:0.3.0.70'
-	implementation 'net.fabricmc:access-widener:1.0.0'
+	implementation 'net.fabricmc:tiny-mappings-parser:0.3.0+build.17'
+	implementation 'net.fabricmc:tiny-remapper:0.3.2'
+	implementation 'net.fabricmc:access-widener:1.0.2'
```

Comment:
These all need updating in the 2 installer json files.

Reply:
resolved in https://github.com/FabricMC/fabric-loader/pull/419/commits/4a458a1af6a23da5aacfe5bdfa64d29cd467e693 and https://github.com/FabricMC/fabric-loader/pull/419/commits/10380818b84e6b0f9b77dab6b19c83417ee0edc5